src/components/navigation/Compass.jsx

```diff
@@ -45,8 +45,8 @@ const Compass = () => {
   } else {
     let quat = new Quaternion(orientX, orientY, orientZ, orientW);
     let rpy = new Euler().fromQuaternion(quat, Euler.ZYX);
-    let attitude = getAttitude(rpy.roll, rpy.pitch);
-
+    alt = getAttitude(rpy.roll, rpy.pitch);
```

Comment:
Shouldn't `alt` should be the altitude from the store without any processing? I think you might be confusing altitude and attitude.

Reply:
Ya, definitely keep attitude around. You can see it's used just a few lines down. Attitude is basically the orientation of the rover (roll, pitch, yaw).